crates/fuel-core/src/service/sub_services/algorithm_updater.rs

```diff
@@ -265,6 +270,9 @@ fn sync_metadata_storage_with_on_chain_storage(
                 metadata_storage,
             )?;
         }
+        AlgorithmUpdater::V1(_) => {
+            todo!()
```

Comment:
Oh and here... when is this to be implemented?

Reply:
addressed in c4ea025dc